social_core/tests/backends/open_id_connect.py

```diff
@@ -228,3 +234,12 @@ def test_invalid_kid(self) -> None:
         self.authtoken_raised(
             "Token error: Signature verification failed", kid="doesnotexist"
         )
+
+    def test_invalid_at_hash(self) -> None:
+        if self.skip_invalid_at_hash:
+            self.skipTest("the call doesn't match any registered mock.")
```

Comment:
This is in because `einfracz`, `elixr` and `fence` do not have responses in place for a successful `do_login`